components/navbar/navbar.jsx

```diff
@@ -483,19 +483,26 @@ export default class Navbar extends React.Component {
                     );
                 } else {
                     addMembersOption = (
-                        <li role='presentation'>
-                            <ToggleModalButton
-                                ref='channelInviteModalButton'
-                                role='menuitem'
-                                dialogType={ChannelInviteModal}
-                                dialogProps={{channel, currentUser: this.state.currentUser}}
-                            >
-                                <FormattedMessage
-                                    id='navbar.addMembers'
-                                    defaultMessage='Add Members'
-                                />
-                            </ToggleModalButton>
-                        </li>
+                        <ChannelPermissionGate
```

Comment:
Wondering why this component doesn’t take care of everything like channelId, teamId, permissions, etc

Obviously non-blocking just curious

Reply:
What do you mean?

Reply:
👍 on that point. I think `teamId` should be handled by ChannelPermissionGate but I'll defer to permission's team on that.
@jespino any comment before I merge this PR?

Reply:
I'll merge this now.